src/main/scala/com/typesafe/sbt/packager/archetypes/JavaServerApplication.scala

```diff
@@ -295,15 +293,44 @@ object JavaServerAppPackaging extends AutoPlugin {
     Some(script)
   }
 
-  protected def rpmScriptletContent(dir: File, script: String,
-    replacements: Seq[(String, String)], definedScript: Option[String], archetype: String = ARCHETYPE, config: Configuration = Rpm): Option[String] = {
-    val file = (dir / script)
-    val template = if (file exists) Some(file.toURI.toURL) else None
+  /**
+   *
+   *
+   * @param scriptDirectory
+   * @param scripts
+   * @param replacements
+   */
+  protected def rpmScriptletContents(scriptDirectory: File, scripts: Map[String, Seq[String]], replacements: Seq[(String, String)]): Map[String, Seq[String]] = {
+    import RpmConstants._
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/nepomukseiler/sbt-native-packager/file/1559303152/issues/source?bid=2040925&fileBranchId=2489531#l304)